pygmt/src/grdgradient.py

```diff
@@ -152,17 +154,14 @@ def grdgradient(grid, **kwargs):
 
     Example
     -------
-    >>> import pygmt  # doctest: +SKIP
+    >>> import pygmt
     >>> # Load a grid of @earth_relief_30m data, with an x-range of 10 to 30,
     >>> # and a y-range of 15 to 25
     >>> grid = pygmt.datasets.load_earth_relief(
     ...     resolution="30m", region=[10, 30, 15, 25]
-    ... )  # doctest: +SKIP
+    ... )
     >>> # Create a new grid from an input grid, set the azimuth to 10 degrees,
-    >>> # and the direction to "c" for Cartesian coordinates
-    >>> new_grid = pygmt.grdgradient(
-    ...     grid=grid, azimuth=10, direction="c"
-    ... )  # doctest: +SKIP
+    >>> new_grid = pygmt.grdgradient(grid=grid, azimuth=10)
```

Comment:
The original example gives the following error:
```
grdgradient [ERROR]: Cannot specify both -A and -D
```
so I update this example.

But I'm using GMT dev branch.